prerelease.sh

```diff
@@ -39,19 +39,3 @@ if [[ ! -z "${files}" ]]; then
 	echo "Boilerplate missing in: ${files}."
 	exit 1
 fi
-
-# Check that cobra docs are up to date
-# This is done by generating new docs and then seeing if they are different than the committed docs
-echo "Checking help documentation..."
-if [[ $(git diff) ]]; then
-  echo "Skipping help text check because the git state is dirty."
-else
-  make gendocs
-  files=$(git diff)
-  if [[ $files ]]; then
-    echo "Help text is out of date: $files \n Please run 'make gendocs'\n and make sure that those doc changes are committed"
-    exit 1
-  else
-    echo "Help text is up to date"
-  fi
-fi
```

Comment:
>  Are we removing this check because we have the docs in doc?

No

> Did we move the check to some other place? 

No

> I considered this check was a good thing.

In a scenario where you build the docs and deploy it on a server, this check does not make any sense. The docs will be build from scratch, so the files are per definition in sync.